.test-infra/mock-apis/infrastructure/kubernetes/refresher/overlays/123079b5-1e58-4b28-a185-66702e2b10c3/deployment.yaml

```diff
@@ -0,0 +1,27 @@
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements. See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License. You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Configures patch for ../base/deployment.yaml
+# See https://kubectl.docs.kubernetes.io/references/kustomize/kustomization/patches/
+
+- op: replace
+  path: /metadata/labels/quota-id
+  value: 123079b5-1e58-4b28-a185-66702e2b10c3
```

Comment:
Where is this value coming from/what does this do? Same question applies across the board for all of these guids in configmap/deployment files

Reply:
It needs an assigned unique identifier. The cleanest way to do this is to simply hard-code the value. The alternative is to create a whole automation around generating a refresher dynamically but it would require the use of a Kubernetes client. I figured that we will have maybe 10 of these so the whole dynamic route didn't seem worth it.